go.mod

```diff
@@ -5,13 +5,15 @@ go 1.19
 require (
 	buf.build/gen/go/bufbuild/knit/bufbuild/connect-go v1.7.0-20230504140941-3dc602456973.1
 	buf.build/gen/go/bufbuild/knit/protocolbuffers/go v1.30.0-20230504140941-3dc602456973.1
-	buf.build/gen/go/bufbuild/reflect/bufbuild/connect-go v1.5.2-20230130220156-b9ed617e60db.1
-	buf.build/gen/go/bufbuild/reflect/protocolbuffers/go v1.28.1-20230130220156-b9ed617e60db.4
+	buf.build/gen/go/bufbuild/reflect/bufbuild/connect-go v1.7.0-20230420145030-e29739ab7137.1
+	github.com/bradfitz/gomemcache v0.0.0-20230124162541-5f7a7d875746
 	github.com/bufbuild/connect-go v1.7.0
 	github.com/bufbuild/connect-grpcreflect-go v1.0.1-0.20230317120624-8e24e9604364
+	github.com/bufbuild/prototransform v0.0.0-20230518140609-c3b595c56388
```

Comment:
This is currently pointing to a off-main commit: https://github.com/bufbuild/prototransform/pull/31

After that PR lands, I'll cut a v0.1.0 release of prototransform and then update this PR to point to the release version.

Reply:
Just pushed a fix for this.